#! /bin/sh /usr/share/dpatch/dpatch-run
##
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes rdoc option passing in a rake file. Upstream bug 3757 and 
## DP: Debian bug 357011.

@DPATCH@
Index: railties/lib/tasks/documentation.rake
===================================================================
--- a/railties/lib/tasks/documentation.rake	(revision 3547)
+++ b/railties/lib/tasks/documentation.rake	(working copy)
@@ -2,7 +2,7 @@
 Rake::RDocTask.new("appdoc") { |rdoc|
   rdoc.rdoc_dir = 'doc/app'
   rdoc.title    = "Rails Application Documentation"
-  rdoc.options << '--line-numbers --inline-source'
+  rdoc.options << '--line-numbers' << '--inline-source'
   rdoc.rdoc_files.include('doc/README_FOR_APP')
   rdoc.rdoc_files.include('app/**/*.rb')
 }
